### PR TITLE
Document enable-puppet4 parameter on our templates

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: docs/installation/clients.md [hunk 1]
```diff
@@ -64,3 +64,7 @@ Now you are ready to install the katello-agent:
 ## Provisioned
 
 In order to install the katello-agent package on a host you are provisioning, you will need to make the appropriate client repository available within your Katello. The first step is to either create a new product or add to an existing product, the appropriate client repository from the dropdown in the [manual](#manual) section above. After you create the new repositories, they will need to be synced locally. Next, you will then need to add them to the relevant content view(s) for the hosts you are wanting to provision. At this point, a new version of the content view can be published and promoted to the appropriate environments that you are wanting to provision a host into. At this point, you can go provision a host and the host will install the katello-agent package during setup.
+
+When provisioning new clients that should use Puppet 4, set a parameter called 'enable-puppet4' set to 'true', so the templates know which package to install and where to place the configuration.  This parameter can be placed at the host, host group, or another appropriate level of the hierarchy.
```
Comment (ehelms) [inline docs/installation/clients.md]: double `set` on the first line

File: docs/upgrade/puppet.md [hunk 2]
```diff
@@ -26,7 +26,10 @@ When planning your upgrade, keep in mind the following compatbility matrix:
 | Puppet 3 Server | Compatible                                     | Not Compatible   |
 | Puppet 4 Server | Compatible (requires conf changes on client)   | Compatible       |
 
-Detailed information about Puppet client support is available in the [Puppet documentation](https://docs.puppet.com/puppet/4.0/reference/upgrade_agent.html).
+
+When provisioning new clients that should use Puppet 4, set a parameter called 'enable-puppet4' set to 'true', so the templates know which package to install and where to place the configuration.  This parameter can be placed at the host, host group, or another appropriate level of the hierarchy.
```
Comment (ehelms) [inline docs/upgrade/puppet.md]: double `set` on the first line

**Comment by ehelms** [comment]
@stbenjam mind updating this so we can get it merged for users?

**Comment by stbenjam** [comment]
Updated

**Comment by ehelms** [comment]
Thanks @stbenjam !

**Pull request merged**